src/test/kotlin/com/autonomousapps/internal/ClassSetReaderTest.kt

```diff
@@ -0,0 +1,30 @@
+package com.autonomousapps.internal
+
+import com.autonomousapps.fixtures.SeattleShelter
+import org.junit.Rule
+import org.junit.rules.TemporaryFolder
+import kotlin.test.Test
+import kotlin.test.assertTrue
+
+class ClassSetReaderTest {
+
+    @get:Rule val tempFolder = TemporaryFolder()
+
+    private val shelter = SeattleShelter()
+
+    @Test fun `jar file analysis is correct`() {
```

Comment:
I think a copy-paste error is here and elsewhere in this test. This is the class-files analysis.